src/content/guides/prepare-azure-account-for-guest-clusters/index.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "Prepare an Azure account to run Giant Swarm guest clusters"
+description = "This guide will walk you through all necessary steps to set up an Azure account with approriate role definition and service principal for operating Giant Swarm guest clusters."
+date = "2018-07-18"
+type = "page"
+weight = 100
+tags = ["tutorial"]
++++
+
+# Prepare an Azure account to run Giant Swarm guest clusters
+
+In a Giant Swarm installation the guest clusters (the clusters running your Kubernetes workloads) can run in a separate Azure subscription (or account) from the host cluster. This gives great flexibility depending on the requirements and the usage scenario. For example, it allows the host cluster to be running in an Azure subscription owned by Giant Swarm, while guest clusters operate in different Azure subscriptions each, depending on a customer’s department using them.
+Giant Swarm operate guest clusters using a service called `azure-operator` which runs on the host cluster.
+
+## Overview
+
+In order to run Giant Swarm guest clusters, an Azure subscription need the following elements :
+
+- Role definition: set of permission to operate guest clusters in the Azure subscription.
+- Service principal: an identity (bound to the previoulsy defined role definition) to access the Azure subscription.
+
+## Create Azure role definition & service principal
+
+In order to perform necessary actions to deploy and maintain guest clusters in your Azure subscription, `azure-operator` needs to access the subscription using a service principal.
+Here we descirbe all the steps to set it up.
+
+1. Prerequisites
+
+To create a service principal you need:
+
+- An account with [Owner](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#owner) or [User Access Administrator](User Access Administrator) role
+- [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) installed
+
+2. Role definition
+
+Download our [role definition](https://raw.githubusercontent.com/giantswarm/azure-operator/38caa99efac9db440433c73646de54a5478f8cb6/policies/guest.json).
```

Comment:
Process question: How will we keep this link up to date?

Reply:
This is not the final link, I just linked the doc in the Pull Request, but I will make sure to merge the PR before this one and change the link to
`https://raw.githubusercontent.com/giantswarm/azure-operator/master/policies/guest.json`